### PR TITLE
Migrate to message-queue-gonsumer v1

### PR DESCRIPTION
**Pull request by mchompalova**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ jobs:
   build:
     working_directory: /go/src/github.com/Financial-Times/post-publication-combiner
     docker:
-    - image: circleci/golang:1.10
+    - image: circleci/golang:1
```
[the remark this answers is not in the document]
Comment (MiroslavGatsanoga) [reply .circleci/config.yml]: +1
The image we should be using is `golang:1` as pointed out [here](https://github.com/Financial-Times/upp-docs/tree/master/guides/howto/update-docker-image-golang).

File: .circleci/config.yml [hunk 2]
```diff
@@ -12,20 +12,21 @@ jobs:
...
     - run:
         name: Go build
-        command: go build -v
+        command: go build -mod=readonly -v
+    - run:
+        name: Run linters
+        command: golangci-lint run --new-from-rev=master --config upp-coding-standard/golangci-config/.golangci.yml
     - run:
         name: Run tests
         command: go test -race -cover -coverprofile=${CIRCLE_COVERAGE_REPORT}/coverage.out ./... | go-junit-report > ${CIRCLE_TEST_REPORTS}/junit.xml
```
Comment (MiroslavGatsanoga) [inline .circleci/config.yml]: It is good to pass `-mod=readonly` to the test command as well.

File: Readme.md [hunk 3]
```diff
@@ -18,13 +18,10 @@ This service depends on the following services:
...
 
-In order to install, execute the following steps:
+Download the source code, dependencies and test dependencies:
 
-        go get -u github.com/Financial-Times/post-publication-combiner
+        go get github.com/Financial-Times/post-publication-combiner
         cd $GOPATH/src/github.com/Financial-Times/post-publication-combiner
-        dep ensure -vendor-only
-        go test ./...
-        go install
```
Comment (MiroslavGatsanoga) [inline Readme.md]: We should leave the `go install`.

**Comment by karsov** [comment]
> Why are we increasing the major version number, are there changes that break compatibility?

@MiroslavGatsanoga The new version of the library does not work with the old version of Kafka.

**Pull request closed without merging**